tg/models/__init__.py

```diff
@@ -58,43 +53,53 @@ def first_chat(self):
         return False
 
     def next_msg(self, step=1):
-        chat_id = self.chats.chat_ids[self.current_chat]
+        chat_id = self.chats.id_by_index(self.current_chat)
         return self.msgs.next_msg(chat_id, step)
 
     def prev_msg(self, step=1):
-        chat_id = self.chats.chat_ids[self.current_chat]
+        chat_id = self.chats.id_by_index(self.current_chat)
         return self.msgs.prev_msg(chat_id, step)
 
     def jump_next_msg(self):
-        chat_id = self.chats.chat_ids[self.current_chat]
+        chat_id = self.chats.id_by_index(self.current_chat)
         return self.msgs.jump_next_msg(chat_id)
 
     def jump_prev_msg(self):
-        chat_id = self.chats.chat_ids[self.current_chat]
+        chat_id = self.chats.id_by_index(self.current_chat)
         return self.msgs.jump_prev_msg(chat_id)
 
     def get_chats(self, offset=0, limit=10):
         return self.chats.fetch_chats(offset=offset, limit=limit)
 
     def send_message(self, text):
-        chat_id = self.get_current_chat_id()
+        chat_id = self.chats.id_by_index(self.current_chat)
         if chat_id is None:
             return False
         self.msgs.send_message(chat_id, text)
         return True
 
     def delete_msg(self):
-        chat_id = self.get_current_chat_id()
+        chat_id = self.chats.id_by_index(self.current_chat)
         if chat_id:
             return self.msgs.delete_msg(chat_id)
 
 
 class ChatModel:
     def __init__(self, tg):
         self.tg = tg
-        self.chats = []  # Dict[int, list]
+        self.chats = []
         self.chat_ids = []
 
+    def id_by_index(self, index):
```

Comment:
potentially bottleneck, so on each operation like go to next chat the app will do `nlogn` operation of sorting chats. Shouldn't be a problem for now, but lets monitor the performance.